### DISCUSSION TITLE
Version 1.0.5

### DISCUSSION THREAD

**Discussion by bengtmartensson** [bengtmartensson/AGirs#68] [General]
Bugfix for #67.

This version uses Infrared4Arduino 1.2.2.

The scripts GirsLite-1.0.5-nano-flasher.sh (Linux) and GirsLite-1.0.5-nano-flasher.bat (Windows) can be used for flashing an Arduino Nano in "canonical configuration" directly, without compiling.

<hr /><em>This discussion was created from the release <a href='https://github.com/bengtmartensson/AGirs/releases/tag/Version-1.0.5'>Version 1.0.5</a>.</em>

**Comment by newmguy** [comment]
Just wondering if this code can be compiled for an atmega8 because there are no atmega328p to be found anywhere. 
I tried but there are just too many errors. I just want to use the TSMP5000 capture mode with IRScrutinizer. No transmit or anything else.
Thanks

**Reply by bengtmartensson** [reply]
AGirs uses [Infrared4Arduino](https://github.com/bengtmartensson/Infrared4Arduino), which does not support ATmega8. It is likely possibly to port at least a subset to ATmega8, but, in particular for the use case you mention, it has just too little memory to be of any practical use.

You can get an "chinese" Nano clone from ebay or Alibaba for a very low price (but note the delivery time). See [this](http://www.harctoolbox.org/arduino_nano.html).

**Comment by newmguy** [comment]
Thank you for responding. Looking at the Infrared4Arduino code I see that it will never work in an atmega8. I will take your suggestion and buy a Nano from China. Thanks
